containers-sig-laws/app/Main.hs

```diff
@@ -0,0 +1,77 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Main where
+
+import           Test.QuickCheck
+import qualified Map.Contrib.Laws.Int as LI
+import qualified Map.Contrib.Laws.Ord as LO
```

Comment:
Currently we have only signatures only for `Map.Strict` from the `containers` package. In future we will add interfaces to `Map.Lazy` as well. So to distinguish both maps the qualified imports will look like `LOS` and `LOL` 😂